code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -1,5 +1,5 @@
 /obj/item/weapon/gun/projectile/automatic/pistol
-	name = "\improper FK-69 pistol"
+	name = "stechkin pistol"

```

Comment:
Because fuck you it was called a stetchkin for at least three years before anyone even realised there was an actual gun called a stetchkin, you /k/ fuck.